tests/test-content/test-content.gresource.xml

```diff
@@ -5,6 +5,10 @@
         <file>backgroundHome</file>
         <file>logo</file>
     </gresource>
+    <gresource prefix="/app">
```

Comment:
It would be better to refactor the lines above so that the image and logo are in an `assets/` directory and you can put the whole thing under `prefix="/app"`.